First webpage/style.css

```diff
@@ -0,0 +1,33 @@
+/* Here in the css file you'll write your styling. make sure to link the css file to the html file. If it's linked correctly you should see a light blue background. */
+
+body {
+  background-color: rgb(180,132,132); width: 1200px; display: block; margin-left: auto; margin-right: auto;
+}
+
+h1 {color: rgb(92,23,30); text-align: center; font-weight:bolder }
+
+h2 {font-weight: bold; text-align: center }
+
+.invisiblebox {width: 800px; margin-left: 150px; margin-right: 150px;} 
+
+.mainbox { background-color: antiquewhite; border-color:	rgb(72,72,72); color: 	rgb(72,72,72);
+  padding-left: 50px; padding-right: 50px; padding-bottom: 10px; padding-top: 10px; 
```

Comment:
When writing css it is customary to write each property and property value on a new line like you do the .row and .column